### PR TITLE
(PA-6502) Update puppet acceptance tests for Rocky9 intel

### PR DESCRIPTION
**Pull request by amitkarsale**
update spec to consider Rocky9 with el9.

### CODE REVIEW

**Comment by mhashizume** [comment]
This will conflict with changes made in https://github.com/puppetlabs/puppetlabs-puppet_agent/pull/724, can you wait until that PR is merged and rebase?

**Review by imaqsood — APPROVED** [review]

**Pull request merged**
